src/base/bittorrent/session.cpp

```diff
@@ -1626,20 +1642,20 @@ bool Session::deleteTorrent(const QString &hash, const bool deleteLocalFiles)
     emit torrentAboutToBeRemoved(torrent);
 
     // Remove it from session
-    if (deleteLocalFiles) {
+    if (deleteOption != Torrent) {
         const QString rootPath = torrent->rootPath(true);
         if (!rootPath.isEmpty())
```

Comment:
not your code but please help us add curly brackets to it and to the `else` cases too.

Reply:
Done.